indico/vendor/django_mail/message.py

```diff
@@ -428,6 +428,7 @@ def _create_attachment(self, filename, content, mimetype=None):
                 filename.encode('ascii')
             except UnicodeEncodeError:
                 filename = ('utf-8', '', filename)
+            attachment.add_header('Content-ID', filename)
```

Comment:
These files cannot be modified, they are 1:1 copies from another project.

Reply:
Maybe something on https://stackoverflow.com/questions/3787755/django-how-to-send-html-emails-with-embedded-images helps

Reply:
perfect, I thought I could escape using MIMEImage

Reply:
BTW, filenames are not guaranteed to be unique so they do not make good cids

Reply:
Yes, they're not unique so I was passing the html_field_name + filename as the new filename which I think should be unique

Reply:
Hi @ThiefMaster, one thing missing from this pr is the display of old image and new image side by side in the case of registration modification as you do for the other fields. Is it necessary to display the previous for images as well?

Reply:
i don't think we need that...

Reply:
okay then.